ddl.dic

```diff
@@ -715,6 +715,125 @@ save_dictionary_audit.version
 
 save_
 
+save_DICTIONARY_AUTHOR
+
+    _definition.id                DICTIONARY_AUTHOR
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-02
+    _description.text
+;
+    The CATEGORY of data items used for dictionary author(s) details.
+;
+    _name.category_id             DICTIONARY
+    _name.object_id               DICTIONARY_AUTHOR
+    _category_key.name            '_dictionary_author.id'
+
+save_
+
+save_dictionary_author.address
```

Comment:
I might be wrong, but I do not predict that the `_dictionary_author.address` will be used much. What is the predicted use case -- sent me mail by post with questions regarding the dictionary and I will get back to you in 2-4 weeks? :smile:

I would rather add the option to add the author's email address (see _publ_author.email).

Reply:
Ah. Yeah, copied from main, not the audit_author extension branch...